EIPS/eip-5115.md

```diff
@@ -0,0 +1,411 @@
+- Yield-generating assets that have different base tokens used for minting vs accounting for the pool value.
+  - This category includes AMM liquidity tokens (which are yield generating assets that yield swap fees) since the value of the pool is measured in “liquidity units” (for example, $\sqrt k$ in UniswapV2, as defined in UniswapV2 whitepaper) which can’t be deposited in (as they are not tokens).
+  - This extends the flexibility in minting the yield-bearing assets. For example, there could be an ETH vault that wants to allow users to deposit cETH directly instead of ETH, for gas efficiency or UX reasons.
+- Assets with reward tokens by default (e.g. COMP rewards for supplying in Compound). The reward tokens are expected to be sold to compound into the same asset.
+
+While EIP-4626 is a well-designed and suitable standard for most vaults, there will inevitably be some yield-generating mechanisms that do not fit into their category (LP tokens for instance). A more flexible standard is required to standardize the interaction with all types of yield generating mechanisms.
+
+Therefore, we are proposing Super Composable Yield (SCY), a flexible standard for wrapped yield generating tokens that could cover most mechanisms in DeFi. We foresee that:
+
+- EIP-4626 will still be a popular vault standard, that most vaults should adopt.
+- SCY tokens can wrap over most yield generating mechanisms in DeFi, including EIP-4626 vaults for projects built on top of interest-bearing tokens.
+- Whoever needs the functionalities of SCY could integrate with the existing SCY tokens or write a new SCY (to wrap over the target interest-bearing token).
+
+### Use Cases
+
+This EIP is designed for flexibility, aiming to accommodate as many yield generating mechanisms as possible. Particularly, this standard aims to be generalized enough that it supports the following uses cases and more:
+
+- Money market supply positions
+    - Lending DAI in Compound, getting DAI interests and COMP rewards
+    - Lending ETH in BenQi, getting ETH interests and QI + AVAX rewards
+    - Lending USDC in Aave, getting USDC interests and stkAAVE rewards
+- AMM liquidity provision
+    - Provide ETH + USDC to ETHUSDC pool in SushiSwap, getting swap fees in more ETH+USDC
+    - Provide ETH + USDC to ETHUSDC pool in SushiSwap and stake it in Sushi Onsen, getting swap fees and SUSHI rewards
+    - Provide USDC+DAI+USDT to 3crv pool and stake it in Convex, getting 3crv swap fees and CRV + CVX rewards
+- Vault positions
+    - Provide ETH into Yearn EIP-4626 vault, where the vault accrues yield from Yearn’s ETH strategy
+    - Provide DAI into Harvest and staking it, getting DAI interests and FARM rewards
+- Liquid staking positions
+    - Holding stETH (in Lido), getting yields in more stETH
+- Liquidity mining programs
+    - Provide USDC in Stargate, getting STG rewards
+    - Provide FEI in Tokemak, getting TOKE rewards
```

Comment:
The spellcheck doesn't like TOKE; could try putting it in backticks or just remove this line.

Reply:
Removed instead.